### ISSUE TITLE
GNOME Online Accounts

### ISSUE THREAD

**Issue by vinyanalista** [kamarada/Linux-Kamarada-GNOME#53]
[GNOME Online Accounts](https://wiki.gnome.org/Projects/GnomeOnlineAccounts) is a single sign-on framework for GNOME. It aims to provide a way for users to setup online accounts to be used by the core system and core applications only. Calendar entries show up in GNOME Shell, e-mail in Evolution, online storages are exposed as GVolumes, and so on.

On Linux Kamarada:

- [Use Evolution to connect to Office 365 on Linux](https://kamarada.github.io/en/2017/09/06/use-evolution-to-connect-to-office-365-on-linux/)
- [Get the most out of GNOME syncing your Google account](https://kamarada.github.io/en/2019/04/10/get-the-most-out-of-gnome-syncing-your-google-account/)

**Comment by vinyanalista** [comment]
Trying to add a Google account to GNOME Online Accounts fails with the following error message:

```
Error writing data to TLS socket: The specified session has been
invalidated for some reason.
```

![goa-bug](https://user-images.githubusercontent.com/1576506/70711319-01570480-1cc0-11ea-8942-5586bedf8fd3.png)

**Comment by vinyanalista** [comment]
Google showed me these:

- clearlinux/distribution#476
- clearlinux/distribution#599

**Comment by vinyanalista** [comment]
Updating the [`glib-networking`](https://software.opensuse.org/package/glib-networking) package, it was possible to add a Google account.

Reference: https://www.reddit.com/r/openSUSE/comments/bmgvzx/error_writing_data_to_tls_socket/eomr3q4/

**Comment by vinyanalista** [comment]
Solved in [15.1-Beta-Build25.1](https://github.com/kamarada/Linux-Kamarada-GNOME/releases/tag/15.1-Beta-Build25.1).
